xline/src/client/mod.rs

```diff
@@ -238,15 +249,33 @@ impl Client {
         Ok(response.into())
     }
 
+    /// Gets an kv client.
+    #[inline]
+    pub fn kv_client(&self) -> KvClient {
```

Comment:
Should we expose these sub clients? We have already exposed more high-level interface such as put, range, delete. I guess the kv_client should be purely internal.

Reply:
So do following sub clients.

Reply:
This is just a temporary client implementation, it will be refactor in the future.